### DISCUSSION TITLE
I wish the eyes (👀) emoji reaction on Issues, Discussions and Pull Requests caused me to follow the thread automagically (🪄)

### DISCUSSION THREAD

**Discussion by brainkim** [community/community#8269] [Other Feature Feedback, Questions, & Ideas]
I regularly use the eyes (👀) [emoji reaction from GitHub](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to note interesting bugs, discussions and memes. I don’t mark everything I see with this emoji, but I do it to mean various things like “you are seen,” “I am watching,” or “Intriguing!”

Independent of the various semantic meanings of the two eyes (👀) emoji, I have a startling product discovery to share. **Every single time I have ever responded to an issue, discussion or pull request with said emoji, I have immediately afterwards subscribed to notifications for the issue, discussion or pull request as well.**

So my feature request is just that we automatically follow issues, discussions and pull requests when the emoji with two eyes reaction is activated. This may change the semantics of the emoji, but as a professional programmer, I think adding a little structure to symbols can be nice sometimes.

**Comment by michaelblyons** [comment]
Maybe for the original post, yes, but for replies to a post I use the eyes to mean that I'm looking at the links included in the reply. I do not necessarily want to follow the thread if I 👀 a reply.

**Comment by openjck** [comment]
I would respectfully prefer if using the eye emoji _did not_ automatically subscribe me to the issue. I use the eye emoji to convey an [awkward stare](https://devonzuegel.com/post/the-silence-is-deafening), something that is sorely missing on social media. Given that we usually receive upvotes from fans and angry comments from assholes, the eyes serve as a nice middle ground, conveying the reaction I would have if someone brought up politics at Thanksgiving dinner. (GitHub's thumbs down emoji is nice, too, but doesn't quite convey the same thing.)
